plugins/woocommerce-admin/client/products/sections/product-inventory-section/manual-stock-section.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { RadioControl } from '@wordpress/components';
+import { useFormContext } from '@woocommerce/components';
+import { Product } from '@woocommerce/data';
+
+export const ManualStockSection: React.FC = () => {
+	const { getInputProps, values } = useFormContext< Product >();
+	const inputProps = getInputProps( 'stock_status' );
+	// These properties cause issues with the RadioControl component.
+	// A fix to form upstream would help if we can identify what type of input is used.
```

Comment:
Would be great to have this suggestion into a new issue not to forget it.

Reply:
Yep!  There's an issue already for this one here - https://github.com/woocommerce/woocommerce/issues/35138